components/base/Typography.tsx

```diff
@@ -207,11 +229,74 @@ export const BlockQuote = ({ children, isDark }: Props) => (
 );
 
 export const InlineCode = ({ children, isDark }: Props) => (
-  <StyledInlineCode className={classnames({ dark: isDark })}>
+  <code css={codeStyle} className={classnames({ dark: isDark })}>
     {children}
-  </StyledInlineCode>
+  </code>
 );
 
 export const Link = ({ text, href }: LinkProps) => (
-  <StyledLink href={href}>{text}</StyledLink>
+  <a css={linkStyles} href={href}>
+    {text}
+  </a>
+);
+
+export const MDXTypographyWrapper = ({ children }: { children: any }) => (
+  <div
+    css={css`
+      padding: 20px;
+
+      @media (${minWidth(BREAKPOINT_IPAD)}) {
+        padding: 20px 40px;
+      }
+
+      > h1,
+      h2,
+      h3,
+      h4,
+      h5,
+      h6 {
+        ${headingStyles.global.styles}
```

Comment:
I'm not entirely sure why, but I had to add `.styles` to get these to work in here. I think it has something to do with styling a non-root element? The only downside I'm aware of is it won't share the same CSS class with other components using these styles which isn't a huge deal, and I didn't want to hold this up since it works.